src/libasr/codegen/asr_to_c_cpp.h

```diff
@@ -849,6 +849,10 @@ R"(#include <stdio.h>
                 src += "\\";
             } else if (s[idx] == '\"') {
                 src += "\"";
+            } else if (s[idx] == '\t') {
```

Comment:
Along with this, you can add `\v` too, it is the `vertical tab` escape sequence in python. Should be an easy addition.
https://stackoverflow.com/questions/3380538/what-is-a-vertical-tab